Files/Strings/en-US/Resources.resw

```diff
@@ -1122,16 +1122,22 @@
   <data name="SettingsPreferencesTerminalApplications.Text" xml:space="preserve">
     <value>Terminal Applications</value>
   </data>
-  <data name="PropertySaveErrorDialogText" xml:space="preserve">
+  <data name="PropertySaveErrorDialog.Text" xml:space="preserve">
     <value>There was an issue saving some properties.</value>
   </data>
-  <data name="PropertySaveErrorDialogRetry" xml:space="preserve">
+  <data name="PropertySaveErrorDialog.Title" xml:space="preserve">
+    <value>Error</value>
+  </data>
+  <data name="PropertySaveErrorDialog.PrimaryButtonText" xml:space="preserve">
     <value>Retry</value>
   </data>
-  <data name="PropertySaveErrorDialogCloseAnyway" xml:space="preserve">
+  <data name="PropertySaveErrorDialog.SecondaryButtonText" xml:space="preserve">
     <value>Close anyway</value>
   </data>
-  <data name="PropertySaveErrorDialogCancel" xml:space="preserve">
+  <data name="PropertySaveErrorDialog.CloseButtonText" xml:space="preserve">
+    <value>Cancel</value>
+  </data>
+  <data name="PropertySaveErrorMessage" xml:space="preserve">
     <value>Cancel</value>
```

Comment:
Is the message meant to be "cancel"?

Reply:
@yaichenbaum Yeah, because it just cancels the attempt to save.

Reply:
I thought the message was meant to say "There was an issue saving some properties..."

Reply:
@yaichenbaum Wait no you're right, I see what you're talking about. Sorry about that.